src/lib/ngsi/Scope.cpp

```diff
@@ -308,6 +316,21 @@ int Scope::fill
     return -1;
   }
 
+
+  //
+  // Georel 'near' only valid for a Point geometry
```

Comment:
At the end, it seems that MongoDB only allows points. Thus, the check stills making sense, although it has been implemented in a different way at the end.

NTC